src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -40,6 +42,14 @@ function install()
     cp "${TEMP_PATH}" "${PLIST_PATH}" || "failed to copy plist"
     cp ./bin/runsvc.sh ./runsvc.sh
     chmod 755 ./runsvc.sh
+
+    echo '
+    {
+        "RunAsService": true,
+        "ServiceName": "'${SVC_NAME}'",
```

Comment:
OK.  This is the only piece of data that's interesting.  This is a shell script and this file is only used as (1) it's existance implies it's a service and (2) info for user - what's the svc name.  As we discussed, this file only needs to contain a string, the name of the svc.